packages/loader/src/resource-deserialize/resources/prefab/PrefabDesign.ts

```diff
@@ -1,8 +1,40 @@
+import type { BackgroundMode } from "@oasis-engine/core";
+import { IRefObject } from "@oasis-engine/core/types/asset/IRefObject";
+import { IColor } from "../mesh/IModelMesh";
+
 export interface IPrefabFile {
   entities: Array<IEntity>;
+  background: {
+    mode: 0;
+    color: { r: 0.3; g: 0.3; b: 0.3; a: 1 };
+    texture: null;
+    sky: null;
+  };
+  ambient: {
+    ambientLight: null;
+    diffuseSolidColor: { r: 0.212; g: 0.227; b: 0.259; a: 1 };
+    diffuseIntensity: 1;
+    specularIntensity: 1;
+  };
 }
```

Comment:
IPrefabFile may not need to contain Background and Ambient

Reply:
done